forem/forem
# Add tooltip to cover image button describing optimal image ratio

## What type of PR is this? (check all applicable)

## Description
This adds a tooltip to the cover image button in the "Rich + Markdown" editor when hovered over: "Use a ratio of 100:42 for best results."

https://user-images.githubusercontent.com/17884966/170611684-de34f301-7465-4d72-85a1-d9fe9ddb6d6f.mp4

It also adds the same tooltip as an inline comment under the cover image text for the "Basic" editor (v1 editor):
![Screenshot](https://user-images.githubusercontent.com/17884966/170615102-b9dc4197-1a32-4cbb-8e37-7bf502c3ac15.png)


## Related Tickets & Documents
Closes https://github.com/forem/forem/issues/17769

## QA Instructions, Screenshots, Recordings
1. Open the editor
2. Hover over the Add Cover Image button
3. See that the tooltip appears

### UI accessibility concerns?
Yes, to be addressed as a separate comment below.

## Added/updated tests?

## [Forem core team only] How will this change be communicated?

## [optional] Are there any post deployment tasks we need to perform?
Nope
